crates/sui-network/build.rs

```diff
@@ -17,26 +18,27 @@ fn main() -> Result<()> {
     };
 
     let codec_path = "mysten_network::codec::BcsCodec";
+    let prost_codec_path = "tonic::codec::ProstCodec";
 
     let validator_service = Service::builder()
         .name("Validator")
         .package("sui.validator")
         .comment("The Validator interface")
         .method(
             Method::builder()
-                .name("transaction")
-                .route_name("Transaction")
-                .input_type("sui_types::transaction::Transaction")
-                .output_type("sui_types::messages_grpc::HandleTransactionResponse")
-                .codec_path(codec_path)
+                .name("submit_transaction")
```

Comment:
Is this a safe change? i.e. both old FNs and new FNs need to work during the rollout period.